Libplanet/Blockchain/BlockChain.Swap.cs

```diff
@@ -62,7 +62,11 @@ public partial class BlockChain<T>
                     const string msg =
                         "A chain cannot be reorged into a heterogeneous chain with a " +
                         "different genesis.";
-                    throw new InvalidGenesisBlockException(msg, Genesis.Hash, other.Genesis.Hash);
+                    var invalidGenesisBlockException = new InvalidGenesisBlockException(
+                        msg, Genesis.Hash, other.Genesis.Hash);
+                    Console.WriteLine(invalidGenesisBlockException.NetworkExpected);
+                    Console.WriteLine(invalidGenesisBlockException.Stored);
+                    throw invalidGenesisBlockException;
```

Comment:
I think these lines went in during debugging, right?

Reply:
yes! gonna modify it.